olive/model/__init__.py

```diff
@@ -466,6 +466,7 @@ def __init__(
         dummy_inputs_func: Union[str, Callable] = None,
         hf_config: Union[Dict[str, Any], HFConfig] = None,
         adapter_path: OLIVE_RESOURCE_ANNOTATIONS = None,
+        is_complete_hf_folder: bool = True,
```

Comment:
>is_complete_hf_folder [](http://example.com/codeflow?start=8&length=21)

could we use hf_config for this case by making it be Union[bool, Dict, HFConfig]?

Reply:
@jambayk